include/rosconsole_bridge/bridge.h

```diff
@@ -37,8 +37,19 @@
 #ifndef ROSCONSOLE_BRIDGE_
 #define ROSCONSOLE_BRIDGE_
 
+#include <ros/macros.h>
```

Comment:
It would be good to declare an explicit dependency in the package manifest on [cpp_common](https://github.com/ros/roscpp_core/blob/53fbad3599116e808d0865b0afbed8078acbbc3c/cpp_common/include/ros/macros.h) which provides this header.

Reply:
will add explicit dependency

Reply:
I also listed it as a `run_depend` since the header is exposed transitively to downstream packages.

Reply:
Thanks! just curious, why is `cpp_common` considered a runtime dependency? I thought we only need this for the header, so it's only build/compile time. not sure what `exposed transitively to downstream packages` means.

Reply:
Please see http://www.ros.org/reps/rep-0140.html#removing-run-depend